### ISSUE TITLE
cmd/go: TestNoteReading test failure (note_test.go)

### ISSUE THREAD

**Issue by kostya-sh** [golang/go#11083]
Using tip (bbc4351)

```
ok  	cmd/fix	0.013s
--- FAIL: TestNoteReading (5.30s)
	note_test.go:43: buildID in hello binary = "75011c52e1681173c8712030a5cebd243923eaf8", want "TestNoteReading-Build-ID"
FAIL
FAIL	cmd/go	11.217s
ok  	cmd/gofmt	0.025s
```

git bisect says:
```
16ebe9f72eee1ccba4e94fb2a79afa0785cb554a is the first bad commit
commit 16ebe9f72eee1ccba4e94fb2a79afa0785cb554a
Author: Russ Cox <rsc@golang.org>
Date:   Thu Jun 4 14:41:02 2015 -0400

    cmd/go: use ELF note instead of binary stamp on ELF systems
```

**Comment by bradfitz** [comment]
Fixed by 48f2d30d4389d57b60afe69cab74a008a2f47047 already

**Comment by kostya-sh** [comment]
That was quick ;)
